.github/workflows/update-browser-pin.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will update the version in .browser and open a PR
+
+name: 'Update pinned Chrome'
+
+on:
+  schedule:
+    # Run hourly at xx:05.
```

Comment:
Isn't `hourly` too much?

`weekly` (to match with dependabot that updates WPT) is better, IMHO.

Reply:
The more frequent the better, as it would allow us knowing about breaking changes early

Reply:
The PR will only be opened/updated if there are changes. The frequency matches the CfT dashboard frequency.